controllers/challenge.js

```diff
@@ -12,14 +12,11 @@ const sendChallengeResponse = async (req, res) => {
   try {
     if (req.method === 'GET') {
       const allChallenges = await challengeQuery.fetchChallenges()
-      if (allChallenges.length > 0) {
-        return res.status(200).json({
-          message: 'Challenges returned successfully!',
-          challenges: allChallenges
-        })
-      } else {
-        return res.boom.notFound('No challenges found')
-      }
+      const challengesWithParticipants = await Promise.all(await getParticipantsofChallenges(allChallenges))
```

Comment:
How about moving the thing inside the bracket to a variable and using that variable here?

Reply:
```JS
const promiseArray = await getParticipantsofChallenges(allChallenges)
const challengesWithParticipants = await Promise.all(promiseArray)
```